ios/AppLovinMAX.m

```diff
@@ -1359,6 +1383,20 @@ - (void)positionAdViewForAdUnitIdentifier:(NSString *)adUnitIdentifier adFormat:
     [NSLayoutConstraint activateConstraints: constraints];
 }
 
+- (void)setPendingExtraParametersIfNeeded:(ALSdkSettings *)settings
+{
+    NSDictionary *extraParameters;
+    
+    if ( _extraParametersToSet.count <= 0 ) return;
```

Comment:
Can we use properties instead of direct access? `self.extraParametersToSet` and `self.sdk`. Unity plugin uses direct instance access (`_`) b/c the fields there are static.

Reply:
My bad...,   updated it, please review again.